packages/devtools_app/lib/src/screens/deep_link_validation/validation_details_view.dart

```diff
@@ -295,12 +296,7 @@ class _GenerateAssetLinksPanel extends StatelessWidget {
                                     ),
                                     IconButton(
```

Comment:
can we use the `CopyToClipboardControl` widget here instead?

Reply:
Yes! updated it.